packages/manager/modules/billing/src/autoRenew/actions/delete/delete.routing.js

```diff
@@ -60,12 +59,8 @@ export default /* @ngInject */ ($stateProvider) => {
           chapter2: 'account',
           chapter3: 'billing',
         });
-        service.setForResiliation();
-        return BillingAutoRenew.updateService({
```

Comment:
i would advise this one needs extensive testing for multiple type of products

Reply:
@frenautvh This has been specified and will go through a proper QA phase